### PR TITLE
Print an error if OpenGL 4.1 isn't supported (cf #56)

### PR DESCRIPTION
**Pull request by PoroCYon**

### CODE REVIEW

**Comment by PoroCYon** [comment]
According to AppVeyor, warnings are critical errors:

```
cmake : CMake Deprecation Warning at external/glfw/CMakeLists.txt:10 (cmake_policy):
At line:1 char:1
+ cmake -DBONZOMATIC_64BIT="$env:x64" -DBONZOMATIC_WINDOWS_FLAVOR:STRIN ...
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : NotSpecified: (CMake Deprecati...(cmake_policy)::String) [], RemoteException
    + FullyQualifiedErrorId : NativeCommandError
 
  The OLD behavior for policy CMP0042 will be removed from a future version
  of CMake.
  The cmake-policies(7) manual explains that the OLD behaviors of all
  policies are deprecated and that a policy should be set to OLD only under
  specific short-term circumstances.  Projects should be ported to the NEW
  behavior and not rely on setting a policy to OLD.
```

**Comment by Gargaj** [comment]
Also sent a note to GLFW about https://github.com/glfw/glfw/issues/1244

File: README.md [hunk 1]
```diff
@@ -16,8 +16,13 @@ The tool was originally conceived and implemented after the Revision 2014 demosc
 - F11 or Ctrl/Cmd-f: hide shader overlay
 - Alt-F4 or Shift+Escape: exbobolate your planet
 
+## Requirements
+On Windows, both DirectX 9 and 11 are supported.
+
+Fr the OpenGL version (for any platform), at least OpenGL 4.1 is required.
```
[the remark this answers is not in the document]
Comment (PoroCYon) [reply README.md]: Fixed.

**Comment by Gargaj** [comment]
GLFW fixed this on their end so perhaps an update is in order (may need @alkama for Mac testing later)

**Comment by alkama** [comment]
Well, considering their fix is making cmake 3.0 mandatory and removing the policy, there is nothing much to "test" :)
Also note that it's on "master" branch, so it'll be coming in 3.3.
I already have a branch using 3.3 nightly, do you want to merge that or wait for their release?

**Pull request merged**